config/setupTests.js

```diff
@@ -8,3 +8,29 @@ global.shallow = shallow;
 global.render = render;
 global.mount = mount;
 global.React = React;
+
+function initFunc() { return; }
+
+function identifyAppFunc() { return; }
+
+function loginFunc() { return; }
```

Comment:
Just a nitpick and since it's test setup no need to change it, but this can be simplified to `const initFunc = () => undeifined;` to me it's visually mor appealing.